packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -0,0 +1,600 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+
+import 'colors.dart';
+import 'icons.dart';
+import 'text_selection.dart';
+
+export 'package:flutter/services.dart' show TextInputType, TextInputAction, TextCapitalization;
+
+const BorderSide _kDefaultRoundedBorderSide = BorderSide(
+  color: CupertinoColors.lightBackgroundGray,
+  style: BorderStyle.solid,
+  width: 0.0,
+);
+const Border _kDefaultRoundedBorder = Border(
+  top: _kDefaultRoundedBorderSide,
+  bottom: _kDefaultRoundedBorderSide,
+  left: _kDefaultRoundedBorderSide,
+  right: _kDefaultRoundedBorderSide,
+);
+const BoxDecoration _kDefaultRoundedBorderDecoration = BoxDecoration(
+  border: _kDefaultRoundedBorder,
+  borderRadius: BorderRadius.all(Radius.circular(4.0)),
+);
+
+const TextStyle _kDefaultTextStyle = TextStyle(
+  fontFamily: '.SF Pro Text',
+  fontSize: 17.0,
+  letterSpacing: -0.38,
```

Comment:
Out of curiosity, where did the letter spacing come from?

Reply:
Good point. Adding comments in code about their provenance.